src/kagglehub/kaggle_cache_resolver.py

```diff
@@ -13,7 +13,7 @@
 KAGGLE_CACHE_MOUNT_FOLDER_ENV_VAR_NAME = "KAGGLE_CACHE_MOUNT_FOLDER"
 ATTACH_DATASOURCE_REQUEST_NAME = "AttachDatasourceUsingJwtRequest"
 
-DEFAULT_KAGGLE_CACHE_MOUNT_FOLDER = "/kaggle/input/"
+DEFAULT_KAGGLE_CACHE_MOUNT_FOLDER = "/kaggle/input"
```

Comment:
A slash is already added below: https://github.com/Kaggle/kagglehub/pull/21/files#diff-524b789095387583a1102b8801823f00e459138589c2cc02bbe6861705306db6R62

Also, in the test environment, the temporary directory created doesn't have a slash at the end (e.g. /tmp/tmpd15pxl5s)